exonum/src/blockchain/tests.rs

```diff
@@ -14,19 +14,63 @@
 
 #![allow(dead_code)]
 
-use std::collections::BTreeMap;
-
 use rand::{thread_rng, Rng};
 use serde_json;
 
-use blockchain::{Blockchain, Schema, Transaction, ExecutionResult};
-use crypto::{gen_keypair, CryptoHash, Hash};
-use storage::{Database, Fork, Error, ListIndex};
-use messages::Message;
+use blockchain::{Blockchain, Service, Snapshot, Schema, Transaction, ExecutionResult};
+use crypto::{gen_keypair, Hash, CryptoHash};
+use storage::{Error, Fork, ListIndex};
+use messages::{Message, RawTransaction};
+use encoding::Error as MessageError;
 use helpers::{Height, ValidatorId};
 
 const IDX_NAME: &'static str = "idx_name";
 
+struct TestService;
+impl Service for TestService {
```

Comment:
I'm insisting on adding an empty line between `TestService` and its `impl` block.